### PR TITLE
README: Add compiler-rt cloning step

### PR DESCRIPTION
**Pull request by JBBgameich**
I hope this is correct, at least it makes it build again for me.

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -21,6 +21,8 @@ You can also use my [fork of gcc](https://github.com/antoyo/gcc) which already i
 ```bash
 $ git clone https://github.com/rust-lang/rustc_codegen_gcc.git
 $ cd rustc_codegen_gcc
+$ git clone https://github.com/llvm/llvm-project llvm --depth 1 --single-branch $
+$ export RUST_COMPILER_RT_ROOT="$PWD/llvm/compiler-rt" $
```
Comment (bjorn3) [inline Readme.md:25]: This shouldn't be needed. compiler-builtins which is built as part of the rust standard library should already provide all necessary intrinsics. What error do you get without?

Comment (bjorn3) [reply Readme.md:25]: Looks like compiler-builtins itself reads this env var if the "c" package feature is enabled, which it shouldn:t at least on x86. What arch do you use?

Comment (JBBgameich) [reply Readme.md:25]: I'm on x86_64. The error I'm getting is
```
error: failed to run custom build command for `compiler_builtins v0.1.67`

Caused by:
  process didn't exit successfully: `/home/jonahb/dev/rust/rustc_codegen_gcc/build_sysroot/target/debug/build/compiler_builtins-8535d9b204370609/build-script-build` (exit status: 101)
  --- stdout
  cargo:rerun-if-changed=build.rs
  cargo:compiler-rt=/home/jonahb/.cargo/registry/src/github.com-1ecc6299db9ec823/compiler_builtins-0.1.67/compiler-rt
  cargo:rustc-cfg=feature="unstable"
  cargo:rustc-cfg=feature="mem-unaligned"

  --- stderr
  thread 'main' panicked at 'RUST_COMPILER_RT_ROOT is not set', /home/jonahb/.cargo/registry/src/github.com-1ecc6299db9ec823/compiler_builtins-0.1.67/build.rs:491:21
  note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace

```

Comment (bjorn3) [reply Readme.md:25]: I'm on x86_64 too. I have never seen this error before. I have no clue why this would happen on x86_64.

Comment (antoyo) [reply Readme.md:25]: That's related to my changes to support non-native integers: some builtins were missing in `compiler-builtins`, so I had to use `compiler-rt` to get them.

Comment (antoyo) [inline Readme.md:25]: I believe the `$` at the end of the lines should be removed.

**Comment by antoyo** [comment]
Thanks!

**Pull request merged**